app/src/main/java/nl/hogeschoolrotterdam/projectb/MemoryDetailActivity.java

```diff
@@ -26,6 +28,8 @@
     Intent shareIntent;
     Memory memory;
     ViewPager2 viewPager2;
+    GoogleMap mGoogleMap;
+    MapView mMapView;
```

Comment:
So, it seems like the only differences in this file (`MemoryDetailActivity`) are 2 new unused variables and a couple empty. Please undo these changes.

Reply:
Seems fixed.